src/units/combine.rs

```diff
@@ -80,22 +94,24 @@ impl Any {
                 };
 
                 match res {
-                    Ok(update) => {
+                    UnitUpdate::Payload(payload) => {
+                        // If it is from our active source, send it on.
+                        // If we don’t have an active source, break out of
+                        // the loop because we now have a candidate.
```

Comment:
I'm not sure why this comment refers to now having a candidate. Maybe simplify this comment to:

```suggestion
                        // the loop.
```

Reply:
I changed it to “because we may now have one” (i.e., an active source).